ledger/src/blockstore_processor.rs

```diff
@@ -1525,10 +1530,35 @@ pub fn confirm_slot(
         load_result
     }?;
 
+    // Process block component markers for Alpenglow slots
+    if let Some(parent_bank) = bank.parent() {
+        if let Some(first_alpenglow_slot) = bank
+            .feature_set
+            .activated_slot(&agave_feature_set::alpenglow::id())
```

Comment:
you'll need to plumb the migration_status and check if is_alpenglow_enabled. Might just be easier to do in _new_from_parent only setting a verifier if it's an alpenglow bank.